index.html

```diff
@@ -44,23 +192,75 @@ <h1 class="flex-row blue-main">I’m Mariane<br>Glad to see you!</h1>
             </a>
           </li>
           <li>
-            <a href="https://medium.com/@Akasha.Rojee" target="_blank" rel="noopener noreferrer">
-              <img src="images/social-icons/medium.png" alt="Medium icon">
+            <a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+              <img src="images/social-icons/angellist.png" alt="AngelList icon">
             </a>
           </li>
           <li>
-            <a href="https://github.com/AkashaRojee" target="_blank" rel="noopener noreferrer">
-              <img src="images/social-icons/github.png" alt="GitHub icon">
+            <a href="https://twitter.com/AkashaRojee" target="_blank" rel="noopener noreferrer">
+              <img src="images/social-icons/twitter.png" alt="Twitter icon">
             </a>
           </li>
           <li>
-            <a href="https://angel.co" target="_blank" rel="noopener noreferrer">
-              <img src="images/social-icons/angellist.png" alt="AngelList icon">
+            <a href="https://medium.com/@Akasha.Rojee" target="_blank" rel="noopener noreferrer">
+              <img src="images/social-icons/medium.png" alt="Medium icon">
             </a>
           </li>
         </ul>
+
+        <div class="flex-row"><button type="button" class="btn font-medium">Get my resume</button></div>
         
       </section>
+
+      <!-- Skills -->
+
+      <section id="skills">
+
+        <div class="skill-card flex-col">
+
+          <div class="skill-category p-medium flex-row align-center justify-between">
+            <div>Languages</div>
+            <div><img src="images/arrow-expanded.png" alt="Click here to collapse"></div>
+          </div>
+
+          <div class="skill-list flex-col">
+            <div class="skill-item flex-row align-center">
+              <img src="images/skills/js.png" alt="JS logo">
+              JavaScript
+            </div>
+            <div class="skill-item flex-row align-center">
+              <img src="images/skills/html.png" alt="HTML logo">
+              HTML
+            </div>
+            <div class="skill-item flex-row align-center">
+              <img src="images/skills/css.png" alt="CSS logo">
+              CSS
+            </div>
+          </div>
+
+        </div>
+
+        <!-- Frameworks -->
+        <div class="skill-card flex-col">
+          <div class="skill-category p-medium flex-row align-center justify-between">
+            <div>Frameworks</div>
+            <div><img src="images/arrow-collapsed.png" alt="Click here to expand"></div>
+          </div>
+          <img src="images/horizontal-separator.png" alt="Separator">
+        </div>
+
+        <!-- Skills -->
+        <div class="skill-card flex-col">
+          <div class="skill-category p-medium flex-row align-center justify-between">
+            <div>Skills</div>
```

Comment:
This text should be bold 
![right](https://user-images.githubusercontent.com/58376020/120894651-d8e06a00-c619-11eb-8c4c-ceb4ad6eb476.png)

Reply:
I have now set the text to bold.